### PR TITLE
♻️🚮 Remove `a4a-doubleclick-disable-sf` experiment

### LINKED ISSUE #13591: Clean up `a4a-doubleclick-disable-sf`
Flag to be enabled only in case of soft rollback of #13594.

Scheduled 2018/3/12.

### PR DESCRIPTION
**Pull request by alanorozco**
The experiment was never enabled so migrating is not necessary.

Fixes #13591

/cc @bradfrizzell @keithwrightbos

### CODE REVIEW

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 1]
```diff
@@ -144,11 +144,18 @@ let sraRequests = null;
       slotId: string,
       slotIndex: string,
     }} */
-let TroubleshootData; // eslint-disable-line no-unused-vars
+let TroubleshootDataDef;
```
Comment (alanorozco) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Drive-by: Renamed this type so that linter won't complain.

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 2]
```diff
@@ -144,11 +144,18 @@ let sraRequests = null;
...
 
 /** @private {?JsonObject} */
 let windowLocationQueryParameters;
 
+
+/**
+ * @typedef
+ * {({width: number, height: number}|../../../src/layout-rect.LayoutRectDef)}
+ */
+let LayoutRectOrDimsDef;
```
Comment (alanorozco) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Defined just for convenience.

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 3]
```diff
@@ -1129,9 +870,22 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
...
+    this.destroySafeFrameApi_();
+    return super.unlayoutCallback();
+  }
+
+  /** @visibleForTesting */
+  cleanupAfterTest() {
+    this.destroySafeFrameApi_();
+  }
+
+  /** @private */
```
Comment (bradfrizzell) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: can you not just make this directly visibleForTesting?

Comment (alanorozco) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: I'd rather expose a private method so it can be used on `unlayoutCallback`. Two entry points :)
